bin/configs.js

```diff
@@ -30,4 +40,25 @@ export const templates = {
       },
     ],
   },
+  express_pg: {
```

Comment:
[node-postgres](https://www.npmjs.com/package/pg) library was expected. Sinice sequelize ORM was used, Change this template name to express_pg_sequelize.